compiler/control/OMROptions.cpp

```diff
@@ -1003,6 +1004,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
         TR::Options::setStaticNumeric, (intptr_t)&OMR::Options::_minProfiledCheckcastFrequency, 0, "F%d", NOT_IN_SUBSET},
    {"minSleepTimeMsForCompThrottling=", "M<nnn>\tLower bound for sleep time during compilation throttling (ms)",
                                          TR::Options::setStaticNumeric, (intptr_t)&OMR::Options::_minSleepTimeMsForCompThrottling, 0, "F%d", NOT_IN_SUBSET },
+   {"moveColdMethodsToColdCode", "M\tmove out-of-line instructions to after last warm instruction", SET_OPTION_BIT(TR_MoveColdMethodsToColdCode), "F"},
```

Comment:
The text describing the option is not correct.

Reply:
Fixed